pkg/vulnloader/nvdloader/convert.go

```diff
@@ -31,6 +31,11 @@ func toJSON(vulns []*apischema.CVEAPIJSON20DefCVEItem) ([]*jsonschema.NVDCVEFeed
 			continue
 		}
 
+		// Ignore rejected vulnerabilities.
+		if strings.EqualFold(vuln.CVE.VulnStatus, "Rejected") {
```

Comment:
@dcaravel @BradLugo I added this conditional after noticing the 2.0 feeds include rejected vulns. They will lack a lot of fields, but just to be safe, I opted to explicitly ignore it

Reply:
I vaguely recall there being some contention with us supporting rejected vulns in the past, and ignoring them as the solution, so your change tracks 👍

Reply:
I didn't add any because they didn't exist before, but that's a bad reason to not add tests. Just added one which covers this rejected case and some basic stuff